File: past/01.js
String.prototype.reverse = function() {
  return this.split('').reverse().join('');
};

var num = 11;

while(true) {
  if((num.toString() == num.toString().reverse()) && (num.toString(2) == num.toString(2).reverse()) && (num.toString(8) == num.toString(8).reverse())){
    break;
  }
  num += 2;
}

console.log(num);

File: past/01.rb
# OpenClass
class Fixnum
  def reverse?(num)
    to_s(num) == to_s(num).reverse
  end
end

num = 11

loop do
  break if num.reverse?(10) && num.reverse?(2) && num.reverse?(8)
  num += 2
end

puts num
